api/v1alpha2/azurecluster_types.go

```diff
@@ -58,7 +58,6 @@ type AzureClusterStatus struct {
 
 // +kubebuilder:object:root=true
 // +kubebuilder:resource:path=azureclusters,scope=Namespaced,categories=cluster-api
-// +kubebuilder:storageversion
```

Comment:
How/why did these get removed?
Same question for `api/v1alpha2/azuremachine_types.go` and `api/v1alpha2/azuremachinetemplate_types.go`.

Reply:
its because the storage version is not v1alpha3, from the [kubebuilder docs](https://book.kubebuilder.io/reference/markers/crd.html)  
```
When conversion is enabled for a CRD (i.e. it‘s not a trivial-versions/single-version CRD), one version is set as the “storage version“ to be stored in etcd. Attempting to store any other version will result in conversion to the storage version via a conversion webhook.

```